doc/hardware/porting/board_porting.rst

```diff
@@ -34,39 +88,26 @@ You can visualize the hierarchy like this:
 Here are some examples. Notice how the SoC series and family levels are
 not always used.
 
-.. list-table::
-   :header-rows: 1
-
-   * - Board
-     - SoC
-     - SoC series
-     - SoC family
-     - CPU core
-     - Architecture
-   * - :ref:`nrf52dk/nrf52832 <nrf52dk_nrf52832>`
-     - nRF52832
-     - nRF52
-     - Nordic nRF5
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`frdm_k64f <frdm_k64f>`
-     - MK64F12
-     - Kinetis K6x
-     - NXP Kinetis
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`stm32h747i_disco <stm32h747i_disco_board>`
-     - STM32H747XI
-     - STM32H7
-     - STMicro STM32
-     - Arm Cortex-M7
-     - Arm
-   * - :ref:`rv32m1_vega_ri5cy <rv32m1_vega>`
-     - RV32M1
-     - (Not used)
-     - (Not used)
-     - RI5CY
-     - RISC-V
+.. table::
+
+   +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
+   | Board                                      | Identifier            | SoC         | SoC Series    | SoC family    | CPU core       | Architecture |
+   +============================================+=======================+=============+===============+===============+================+==============+
+   | :ref:`nrf52dk <nrf52dk_nrf52832>`          | /nrf52832             | nRF52832    | nRF52         | Nordic nRF5   | Arm Cortex-M4  | Arm          |
+   +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`frdm_k64f <frdm_k64f>`               | /mk64f12              | MK64F12     | Kinetis K6x   | NXP Kinetis   | Arm Cortex-M4  | Arm          |
+   +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`rv32m1_vega <rv32m1_vega>`           | /openisa_rv32m1/ri5cy | RV32M1      | (Not used)    | (Not used)    | RI5CY          | RISC-V       |
+   +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`nrf5340dk <nrf5340dk_nrf5340>`       | /nrf5340/cpuapp       | nRF5340     | nRF53         | Nordic nRF5   | Arm Cortex-M33 | Arm          |
```

Comment:
Adding variant would be a useful example here too

Reply:
If I understand the current table correctly it tries to describe how hardware is modeled in Zephyr.

In that context I don't feel variant (or revision) fits in, as those wouldn't change anything in relation to the SoC, SoC series / family, CPU core, architecture.